### PR TITLE
#10 added test harness and got queue to process. yay!

### PR DESCRIPTION
**Pull request by twilson63**
* Got harness working! Yay!
* Created some examples in Readme
* Explained options with was credentials.

### CODE REVIEW

File: mod.js [hunk 1]
```diff
@@ -16,7 +18,10 @@ export default function sqsAdapter(svcName, options = {}) {
   const setName = assoc("name", __, {});
   const createFactory = over(
     lensProp("factory"),
-    () => new ApiFactory({ credentials: options }),
+    () =>
+      (options.awsAccessKeyId && options.awsSecretKey && options.region)
```
Comment (TillaTheHun0) [inline mod.js]: The `README` claims the default region is `us-east-1`, but I don't see a default anywhere. Is that doc still accurate? I did [this](https://github.com/hyper63/hyper-adapter-s3/blob/7c4515a48d2f28d497725b1d16b6b4d228cc60b5/mod.js#L18) on the s3 adapter, albeit a bit more imperative.

File: test/harness.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  colors,
+  core,
+  uniqueNamesGenerator,
+} from "../deps_dev.js";
+import { default as myAdapter, PORT } from "../mod.js";
+
+const name = uniqueNamesGenerator({
+  separator: "-",
+  dictionaries: [adjectives, colors, animals],
+});
```
Comment (TillaTheHun0) [inline test/harness.js:17]: Fun :)

File: mod.js (hunk 1 above)
Comment (twilson63) [reply mod.js]: Ok, added merge to set defaults, if region is not specified but key and secret are.

**Review by TillaTheHun0 — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**